README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
+When capturing a bank account the flow can be divided in two steps:
+
+- Capturing
+- Verification
+
+Once the `Capture` has been made, `stripe` will deposit two random amounts on the bank account. These amounts have to be
+verified against (or in other words, *sent to*) `stripe`. You can do the verification process directly using `billforward-js`.
+
+In more details what happens is:
```

Comment:
This flow is a bit confusing.

I see here that there are two separate flows:
- capture card
- verify card

But this explanation is trying to explain many things at once.

- It is unclear that there are two flows in here.
- It is unclear which steps are actions required of the user.

I think this flow is trying to conflate "what should the user do" with "what will the system do". Perhaps describe those separately; the user should only think about: "what do I input? what are the possible outputs?"